ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -176,7 +176,18 @@ public void onForkChoiceUpdatedResult(
                               .getHeadBlockRoot(),
                           forkChoiceUpdatedResult.getPayloadStatus());
                     }))
-        .finish(error -> LOG.error("Failed to update fork choice", error));
+        .finish(
+            error -> {
+              final String errorMessage = "Failed to update fork choice. ";
+              if (error
+                  .getMessage()
+                  .contains(
```

Comment:
we should probably be looking at the type of exception much like the last one ...

Reply:
Could do that. However, it would require adding a dependency on `teku.ethereum.executionclient.main`. I guess that should be okay.

Reply:
I'd have to take a closer look... there's throwables and another library in infrastructure I thought

Reply:
`ExceptionUtil` is in infrastructure, so if we need to include it we're fine to do that...